### DISCUSSION TITLE
stream info & the admin panel

### DISCUSSION THREAD

**Discussion by compufox** [owncast/owncast#609] [Ideas]
just started playing around with the project and i think its really cool!

on that note, I do wish that I didnt have to open the markdown/etc files when i needed to make a change.

doing it through the admin panel would be much preferred.

related: maybe moving the stream tags/title from the config.yaml into some markdown/separate file would be better.
i know that many users (myself included) who come from twitch/etc and dynamically change their stream titles based on what they're playing that day. having to take the entire server down and back up to change that is a bit of a hassle imo.

**Comment by graywolf336** [comment]
Thank you for opening a discussion and letting us know there is more interest in what we have planned. Actually, this is something that is actively being worked on! You can follow along with the progress via the milestone and let us know your thoughts on what we have planned so far. :)

https://github.com/owncast/owncast/milestone/4

**Reply by compufox** [reply]
Ah! Thank you! I figured something like this had been mentioned, but I didnt know where to look, lol!
